tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -494,7 +494,7 @@ bundle edit_line ncf_maintain_keys_values(v, sep) {
         expression => regline("^${keypat[${index}]}.*", "${edit.filename}");
 
   insert_lines:
-      "${index}${sep}$i{${v}[${index}]}"
```

Comment:
this worked because incorrect line was added, then corrected just below by the replace_patterns